helm/private/helm_install.bzl

```diff
@@ -2,9 +2,36 @@
 
 load("//helm:providers.bzl", "HelmPackageInfo")
 
+def _stamp_opts(ctx, name, opts):
+    opts_file = ctx.actions.declare_file("{}.{}.opts".format(ctx.label.name, name))
+    ctx.actions.write(
+        opts_file,
+        "\n".join(opts),
+    )
+
+    opts_stamped_file = ctx.actions.declare_file("{}.{}.opts.stamped".format(ctx.label.name, name))
+
+    args = ctx.actions.args()
+    args.add("-volatile_status_file", ctx.version_file)
+    args.add("-stable_status_file", ctx.info_file)
+    args.add("-input_file", opts_file)
+    args.add("-output_file", opts_stamped_file)
+
+    ctx.actions.run(
```

Comment:
I think this stamping needs to happen in other actions. Won't this file be out of date if volatile data changed and then this output was now an input to a downstream rule?

Reply:
Ok, moved into the helm runner